compiler/rustc_target/src/asm/mod.rs

```diff
@@ -156,6 +156,7 @@ mod powerpc;
 mod riscv;
 mod spirv;
 mod wasm;
+mod xtensa;
```

Comment:
These modules are listed (sorted) alphabetically, please put "xtensa" after "x86" just to keep it sorted!